### PR TITLE
Add a tip describing how to build a specific commit.

### PR DESCRIPTION
**Pull request by bbewan**
https://app.asana.com/0/353548986073982/425415531859615

### CODE REVIEW

**Comment by bbdocs** [comment]
See the website here: http://69-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://69-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://69-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://69-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://69-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://69-bb-docs.surge.sh

**Comment by bbdocs** [comment]
See the website here: http://69-bb-docs.surge.sh

File: builds/faq.adoc [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+the link:../troubleshooting/README.adoc[Troubleshooting] section for
+solutions to a variety of build problems.
+
+== How do I build a specific commit?
+
+- Create a branch pointing to the specific commit. Then use the
+  link:https://apidocs.buddybuild.com/builds/post-trigger.html[REST API]
+  to trigger a build for that branch.
+
+- Push a tag the includes the desired commit.
```
Comment (Palleas) [inline builds/faq.adoc]: Typo?

File: builds/faq.adoc [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+While buddybuild tracks the history and feedback for all builds, you can
+make builds for old branches less visible:
+
+- Delete the branch if you no longer need it. When you have deleted
+  branches, a **deleted branches** link appears at the bottom of the
+  **Builds** page in the Dashboard; click that link to view the builds
+  for deleted branches.
+
+- Create additional branches; new branches push old branches out of
```
Comment (Palleas) [inline builds/faq.adoc]: Seems like a strange tip to give to people?

**Comment by bbdocs** [comment]
See the website here: http://69-bb-docs.surge.sh

**Pull request merged**